src/main/java/lotto/Domain/LottoSystem.java

```diff
@@ -0,0 +1,90 @@
+package lotto.Domain;
+
+import static lotto.Util.InputValidator.checkDistinctBetweenWinningAndBonusNumber;
+import static lotto.Util.InputValidator.checkDistinctNumbers;
+import static lotto.Util.InputValidator.checkSixNumber;
+import static lotto.Util.InputValidator.isValidRangeNumber;
+import static lotto.Util.TypeCaster.convertStringToIntegerList;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class LottoSystem {
+    private List<Integer> lottoWinningNumbers;
+    private int lottoBonusNumber;
+
+    public void storeLottoWinningNumber(String lottoWinningNumber) {
+        lottoWinningNumber = lottoWinningNumber.replaceAll("\\s*,\\s*", ",");
+        if (isValidRangeNumber(lottoWinningNumber)) {
+            checkSixNumber(lottoWinningNumber);
+            checkDistinctNumbers(lottoWinningNumber);
+        }
+
+        this.lottoWinningNumbers = convertStringToIntegerList(lottoWinningNumber);
+        Collections.sort(this.lottoWinningNumbers);
+    }
+
+    public void storeLottoBonusNumber(String lottoBonusNumber) {
+        lottoBonusNumber.trim();
+        if (isValidRangeNumber(lottoBonusNumber)) {
+            checkDistinctBetweenWinningAndBonusNumber(lottoWinningNumbers, lottoBonusNumber);
+            this.lottoBonusNumber = Integer.parseInt(lottoBonusNumber);
+        }
+    }
```

Comment:
저도 고민했던 부분인데 민서님은 문자열 형태로 넘겨주어서 검증해주는 방식을 택하셨군요!

저도 처음에는 이 방법을 사용했었습니다! 
그런데 검증을 해줄 때 마다 숫자형으로 변환해주거나 숫자형인지 확인해주는 작업이 중복되어서 사용해줘야하더라고요!
```suggestion
    public void storeLottoBonusNumber(String lottoBonusNumberString) {
        validateNumber(lottoBonusNumberString);
        int lottoBonusNumber = Integer.parseInt(lottoBonusNumberString);
        if (isValidRangeNumber(lottoBonusNumber)) {
            checkDistinctBetweenWinningAndBonusNumber(lottoWinningNumbers, lottoBonusNumber);
            this.lottoBonusNumber = Integer.parseInt(lottoBonusNumber);
        }
    }
```
그래서 저는 위 코드와 같이 숫자형 검증 이후에 숫자형으로 변환해줘서 나머지 검증을 하는 방식을 택했습니다! ㅎㅎ
아니면 애초에 `InputView` 에서  숫자형 검증 이후 숫자를 리턴해주는 방식을 사용했어요! 
중간 중간에 `lottoBonusNumber` 가 문자열인지 숫자인지 헷갈리지 않게 처음부터 숫자로 보내주는 방식을 택하니까 코드도 간단해지더라고요! 

물론 제가 말한 것이 정답은 아닙니다! ㅎㅎ 
단지 저는 이렇게 했다고 말씀드리고 싶었어요! 
민서님이 편하신 방법을 택하시면 좋을 것 같습니다!😊

Reply:
@jhon3242 
InputView에서 숫자가 검증될 경우 숫자를 리턴해준다는 것은 
"숫자로 만들어서" 리턴해준다는 의미로 이해했는데 맞을까요?
물론 너무 편할 것 같습니다만 만약 그렇다면 InputView가 하는일이 더 늘어난다고 생각해서 일부러 지양했던 것 같아요.
이부분에 대해서는 어떻게 생각하시나요??

Reply:
너무 좋은 의견입니다! 맞아요! 저도 "InputView 가 입력에 대한 책임 + 검증에 대한 책임을 가지는게 아닌가?" 라고 고민해봤었어요! 그런데 우테코 5기 분이 이전에 대답해주신 답변이 있어서 @m2nseop  님에게도 도움이 될까 하여 공유해봅니다!
<img width="808" alt="image" src="https://github.com/woowacourse-precourse/java-lotto-6/assets/78288539/9c12afed-8d44-4a83-a5f1-e53e76ea7fa7">